src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -607,13 +607,17 @@ def get_assay_name_info(
             for ga in target_assays:
                 if ga["name"] == assay_name:
                     selected_assay_id = ga["uuid"]
-
-    selected_ref_genome = "GRCh38.92"
     
     if friendly_assay_type == "germline":
+        selected_ref_genome = "GRCh38.92"
         for a in target_assays:
             if a["name"] == selected_assay_name and a["reference_genome"]:
-                selected_ref_genome = a["reference_genome"]["name"]
+                selected_ref_genome = a["reference_genome"]["name"].split(".", 1)[1]
```

Comment:
@Niranjanpandeshwar is it enforced that the `a["reference_genome"]["name"]` will always have the subversion suffix `.xxx`? If not this would fail on non-existent index `1`.

Reply:
Yes, either `reference_genome` will be empty, if not, then the name will have either GRCh38.92 or GRCh37.87. These are the only possibilities right now.